roles/passenger/templates/geotaste_passenger.conf.j2

```diff
@@ -0,0 +1,106 @@
+server {
+  listen {{ passenger_listen_port }} default_server;
+  server_name {{ passenger_server_name }};
+
+  # define django environment for split settings
+  passenger_env_var DJANGO_ENV {% if qa is defined %}staging{% else %}production{% endif %};
+
+  set_real_ip_from 128.112.203.144;
+  set_real_ip_from 128.112.203.145;
+  set_real_ip_from 128.112.203.146;
+  real_ip_header X-Real-IP;
+
+  access_log /var/log/nginx/access.log custom;
+
+  # increase upload size from nginx default of 1MB
+  client_max_body_size 0;
+
+  # set landing page
+  root /srv/www/root/html;
+  index index.html;
+  expires -1;
+
+  # Enable Gzip
+  gzip  on;
+  gzip_http_version 1.0;
+  gzip_comp_level 2;
+  gzip_min_length 1100;
+  gzip_buffers     4 8k;
+  gzip_proxied any;
+  gzip_types
+    # text/html is always compressed by HttpGzipModule
+    text/css
+    text/javascript
+    text/xml
+    text/plain
+    text/x-component
+    application/javascript
+    application/json
+    application/xml
+    application/rss+xml
+    font/truetype
+    font/opentype
+    application/vnd.ms-fontobject
+    image/svg+xml svg svgz;
+
+  gzip_static on;
+
+  gzip_proxied        expired no-cache no-store private auth;
+  gzip_disable        "MSIE [1-6]\.";
+  gzip_vary           on;
+
+  {% block root %}
+  location /sco/1 {
+    auth_basic "Restricted Content";
+    auth_basic_user_file /etc/nginx/.htpasswd;
```

Comment:
If we were doing auth, we'd want the password file to be stored in ansible vault and put on the server as part of the playbook.  I'd also want to figure out a way that it would only prompt for the password when you aren't on vpn, but that doesn't matter now.

let's remove these lines.